src/app/engineering/hackathons/page.tsx

```diff
@@ -1,10 +1,10 @@
 import Hackathons from "@/components/engineering/hackathons/Hackathons";
```

Comment:
this should also be similar to the biog page where it just imports and exports the component, no need for an extra div